### ISSUE TITLE
::guilds command 

### ISSUE THREAD

**Issue by busybox11** [ArtiskOnGit/Mayoshi#2]
The `::guilds` command shows all bot's guilds with multiple messages, so in a non-efficient way
A better way to do it would be by sending one big message or an embed

**Comment by ArtiskOnGit** [comment]
true, as this command is only available for admins of mayoshi, not a big deal for now
